docs/UserGuide.adoc

```diff
@@ -467,12 +467,12 @@ A graph is shown.
 // end::graph[]
 
 === Adding a reminder: `reminder add`
-Adds a reminder to the reminder list when user enters a new reminder. Deadline and reminder message are recorded into storage.
+Adds a reminder to the reminder list when user enters a new reminder. This command records a reminder together with its message and deadline.
```

Comment:
> Adds a reminder to the reminder list. Key information such as deadline and message are recorded.

Please change this line to the above so that the description is clearer and to avoid repetition. It should match the format input sequence being described.